### ISSUE TITLE
RedisClient.close() logs WARN on no-op

### ISSUE THREAD

**Issue by mzapletal** [mp911de/lettuce#181]
`RedisClient.close()` logs a WARN message if the client is already closed (https://github.com/mp911de/lettuce/blob/master/src/main/java/com/lambdaworks/redis/RedisChannelHandler.java#L71). Wouldn't it be enough to just log this on INFO in order to avoid log pollution?

**Comment by mp911de** [comment]
Hi @mzapletal 
these are two issues in this ticket. The one is the WARN message. The other one is that you run into the WARN message. You should check from your side, why a connection is closed more then once. That's, in general, a good opportunity to streamline resource disposal and rethink the logging at all.

The some clients (MySQL, MongoDB) just no-op silently while others (Hibernate) throw Exceptions on duplicate close calls.

**Comment by mzapletal** [comment]
Hi Marc, you are right - we had redundant `close()` operations resulting in a bunch of WARN messages. I gonna avoid them now by calling `isOpen()` first (we had to deactivate `autoReconnect` some time ago). I just looked into the code of `RedisClient.close()` and saw that it is returning anyway when `closed` is already true, so I thought logging this at WARN is not really necessary. But I don't mind to leave it like it is ;)
